docker/default/vocascan.config.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * For more help with the file see https://docs.vocascan.com/#/vocascan-server/configuration
+ * This config file lists all available options
+ */
+
+module.exports = {
+  debug: false,
+
+  server: {
+    port: 8000,
+    jwt_secret: '',
+    salt_rounds: 10,
+  },
+
+  database: {
+    dialect: 'postgres',
+    host: 'db',
+    port: '5432',
+    username: 'vocascan',
+    password: 'vocascan',
```

Comment:
Please remove the password from the default config

Reply:
Why should I remove it? This shouldn’t be a security risk because the database is only accessible from inside the docker network. Any client outside can’t access the database. This only provides a better setup experience to new users. And if they want, they can change the password. (The old docker compose file also used vocascan as a default password)